buildmaster/ATLAS_2JET_7TEV_R06/filter.py

```diff
@@ -0,0 +1,329 @@
+# Filter for ATLAS_2JET_7TEV_R06
+"""
+Created on Mar  2023
+
+@author: Mark N. Costantini
+"""
+
+import yaml
```

Comment:
As a general note, I'd prefer using the ruamel.yaml library (or `from reportengine.compat import yaml`).

It doesn't matter very much if you want parse into a simple dict, but it is better to use only one yaml library in the whole codebase, and also the capability to know about things like line numbers and comments is useful for things like this https://validobj.readthedocs.io/en/latest/examples.html#yaml-line-numbers

Reply:
Relatedly, I wonder if the processing of the hepdata raw files could be done with validobj (like we already do for the commondata new format). May or may not be simpler, but very likely more reusable.